pkg/package_archive/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: package_archive
+publish_to: none
+
```

Comment:
Please give this a description and maybe a `README` that just says what this package does..